server/static/js/index/home_page_processing.js

```diff
@@ -0,0 +1,122 @@
+"use strict";
+const TABLE_USERS = '#table_users';
+const TABLE_DEVICES = '#table_device';
+let renderTableDelay = 200;
+
+// Callback function, called on success, for reading device data
+function drawTableDevices (data) {
+    for (let index = 0; index < data.devices.length; index++) {
+        drawRowDevice(data.devices[index]);
+    }
+}
+
+// The function of drawing the rows of the table of devices
+function drawRowDevice (rowDataDevice) {
+    let $row = $('<tr class="table-row table-remove-row-device">');
+    let onlineStatus = rowDataDevice.online;
+    let deviceTypeNumber = rowDataDevice.device_type;
+    let deviceType = {
+        1: "Orange Pi Lite",
+        2: "Orange Pi Zero",
+        3: "Orange Pi One",
+        4: "Orange Pi PC",
+        5: "Orange Pi Plus",
+        6: "Orange Pi 2G-IoT"
+    };
+
+    onlineStatus = onlineStatus === true ? "Online" : "Offline";
+
+    $(TABLE_DEVICES).append($row);
+    $row.append($("<input data-device-id type=\"hidden\">").val(rowDataDevice.id));
+    $row.append($("<td data-device-type-id>" + rowDataDevice.device_id + "</td>"));
+    $row.append($("<td data-device-type>" + deviceType[deviceTypeNumber] + "</td>"));
+    $row.children('[data-device-type]').attr('data-device-type', rowDataDevice.device_type);
+    $row.append($("<td data-device-last-address>" + rowDataDevice.last_address + "</td>"));
+    $row.append($("<td data-device-last-online>" + rowDataDevice.last_connected_at + "</td>"));
+    if (onlineStatus === "Online") {
+        $row.append($("<td data-device-status class='online'>" + onlineStatus + "</td>"));
+    } else {
+        $row.append($("<td data-device-status class='offline'>" + onlineStatus + "</td>"));
+    }
+}
+
+// Callback function, called on success, for reading users data
+function drawTableUsers (data) {
+    for (let index = 0; index < data.users.length ; index++) {
+        drawRowUsers(data.users[index]);
+    }
+}
+
+// The function of drawing user table rows
```

Comment:
Function declarations should have proper JS docstring (if applicable), so consider changing broad comment to a docstring.